apps/web/src/views/CakeStaking/components/Tooltips.tsx

```diff
@@ -29,3 +29,12 @@ export const Tooltips: React.FC<React.PropsWithChildren<TooltipsProps>> = ({
     </>
   )
 }
+
+export const DebugTooltips: React.FC<React.PropsWithChildren<TooltipsProps>> = ({ ...props }) => {
+  return (
+    <Tooltips
+      {...props}
+      disabled={!(window.location.hostname === 'localhost' || process.env.NEXT_PUBLIC_VERCEL_ENV === 'preview')}
```

Comment:
How about replace localhost to node env?

Reply:
> production: When your application is built with next build
> development: When your application is run with next dev

I think node env is not suitable